### ISSUE TITLE
Library Click Events not being triggered

### ISSUE THREAD

**Issue by remicoges** [bassaer/ChatMessageView#77]
Hello,

I have a ChatMessageView and I am not able to get the click events being triggered 

My code : 

`mChatView.setOnBubbleLongClickListener(new Message.OnBubbleLongClickListener() {
            @Override
            public void onLongClick(Message message) {
                System.out.println(message.getMessageText());
            }
        });`

Despite the Icon & Bubble events not being triggered, the options & send button are fine

Is that a problem with my code ?

Thanks in advance for the reply

Rémy

**Comment by bassaer** [comment]
Did you get any errors?

How about using Toast instead of System.out.println?
```
mChatView.setOnBubbleLongClickListener(new Message.OnBubbleLongClickListener() {
            @Override
            public void onLongClick(Message message) {
                Toast.makeText(context, message.getMessageText(),Toast.LENGTH_SHORT).show();
            }
});
```

**Comment by remicoges** [comment]
I haven't got any errors but nothing happens.

Same problem with your Toast snippet

By the way, I am using 1.3.5

**Comment by bassaer** [comment]
sample code works at 1.6.0
https://github.com/bassaer/ChatMessageView/blob/master/example/src/main/java/com/github/bassaer/example/MessengerActivity.java#L113-L118

I cannot find out unless you show your code.

**Comment by arthurabreu** [comment]
There is a bug happening with me.

My code provides a LIST for the chatview to show on screen. Everything is shown alright.

When I click the Bubble for the first time, nothing happens.
If I go back one screen and back to the chatview and click, nothing happens too.

Then, when I send a message, go back one page (which doesn't use the library or the chatview) then load the chatview screen again AND I click on the bubble two times, it works, with a Toast or whatever I want.

I've debugged my code and it's alright.

TL;DR: Need 2 clicks to make the OnBubbleClick to work on the chatview and it only works after I send a message and go back one screen.

Hopefuly I'm being thorough on my description.
